styleguide/pane.scss

```diff
@@ -7,6 +7,11 @@
 $pane-margin: 30vh;
 $easeOutExpo: cubic-bezier(.190, 1.000, .220, 1.000);
 
+.hide,
+.kiln-pane-template-elements {
+  display: none !important; // Always hide your skeletons and other hidden things
```

Comment:
`.hide` is a SUPER generic class name that will probably be used by other things, causing bad side effects.